examples/ProductsApi/delete.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+use Printful\Exceptions\PrintfulApiException;
+use Printful\Exceptions\PrintfulException;
+use Printful\PrintfulApiClient;
+use Printful\PrintfulProductsApi;
+
+require_once __DIR__ . '../../vendor/autoload.php';
```

Comment:
require feilo, jo __DIR__ nav trailing slash. Šis attiecas uz visiem examples failiem, arī tiem, kas nav šajā MR.

Reply:
Saliku trailing slash savos example